test/framework/convenience.go

```diff
@@ -39,8 +39,8 @@ func InstallComponents(ctx context.Context, mgmt Applier, components ...Componen
 		for _, component := range components {
 			By(fmt.Sprintf("installing %s", component.GetName()))
 			c, err := component.Manifests(ctx)
-			Expect(err).NotTo(HaveOccurred())
-			Expect(mgmt.Apply(ctx, c)).To(Succeed())
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
```

Comment:
This method appears to be called by InitManagementCluster, so we should likely handle this a bit better, because this would point to that helper if it fails rather than the caller.

I'm wondering if we can leverage the passed in `context.Context` to help us here. For example:

```go
const ContextOffsetKey = "offset"

func ICallAnotherHelper(ctx context.Context, ...) {
	offset := offsetFromContext + 1
	...
	InstallComponents(context.WithValue(ctx, ContextOffsetKey, offset)...)
}

func InstallComponents(ctx context.Context, ...) {
	offset := offsetFromContext + 1
	...
	ExpectWithOffset(offset, ...)...
}

func offsetFromContext(ctx context.Context) int {
	if offset, ok := ctx.Value(ContextOffsetKey).(int); ok {
		return offset
	}
	return 0
}
```

@chuckha thoughts?